CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,56 @@
+# Code of Conduct
+
+## Our Pledge
+
+We, Erutidians' Core Members, as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or advances of any kind
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email address, without their explicit permission
+- Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+
+All Core Members are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+
+All Core Members have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily/permanently, with properly communicated reasons for moderation decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the Core Members responsible for enforcement at our official [Discord Server](https://discord.gg/CsYcA7m67b) by contacting an admin in [#report](https://discord.com/channels/) channel, or email `report@erutidians.tech`. All complaints will be reviewed and investigated promptly and fairly.
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the Core Members responsible for enforcement at our official [Discord Server](https://discord.gg/5vatpedtsq) by contacting an admin in [#report](https://discord.com/channels/972258637378625597/996868532430184488) channel, or email `report@erutidians.tech`. All complaints will be reviewed and investigated promptly and fairly.
```